gsa/src/gmp/commands/reports.js

```diff
@@ -59,6 +59,16 @@ class ReportsCommand extends EntitiesCommand {
       filter,
     });
   }
+
+  get(params, options) {
+    return super.get(
+      {
+        details: 0, // ensure to request no details by default
+        ...params,
+      },
```

Comment:
Changing the order of these two lines would actually ensure details: 0. In this order, details might be overwritten if they are set in params. I'm not 100% sure, wat we want here...

Reply:
the api may decide to request reports with details like `gmp.reports.get({details: 1, filter=...});`

Reply:
In that case it's the wrong order in gmp.results.get() ?

Reply:
yes you are right. gmp.results.get doesn't allow for not getting the details. IMHO it should be changed.

Reply:
Ok, would you like to put it here as well, or do we need another PR for that?

Reply:
Another PR because it isn't related